var/spack/repos/builtin/packages/vecgeom/package.py

```diff
@@ -19,11 +19,17 @@ class Vecgeom(CMakePackage, CudaPackage):
 
     maintainers("drbenmorgan", "sethrj")
 
-    version("master", branch="master")
+    version("master", branch="master", get_full_repo=True)
```

Comment:
Do you want `get_full_repo=True` package-wide so it applies also to 2.0.0-surfacedev.1?

Reply:
No, it actually doesn't need it since it's a tag. The full repo is only needed whenever the last tag is somewhere before HEAD.